woocommerce/woocommerce
# Version bump: Add README and Logger

### All Submissions:






### Changes proposed in this Pull Request:



Contributes towards https://github.com/woocommerce/woocommerce/issues/34424

Add a README to the version tool as well as make use of the Logger core-cli helper. I also noticed an instance of a relative path, so I fixed that to be more robust.

### How to test the changes in this Pull Request:

1. Bump a version `pnpm run version --filter version-bump -- bump woocommerce -v 7.1.0` and confirm its all still working.

### Other information:




### FOR PR REVIEWER ONLY:
